package.json

```diff
@@ -21,9 +22,12 @@
     "@babel/core": "^7.4.0",
     "@babel/preset-env": "^7.4.2",
     "babel-jest": "^24.5.0",
+    "eslint": "^5.16.0",
```

Comment:
Modify caret to tilda. More details here why: https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json